File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md

```diff
@@ -8,50 +8,52 @@ documentation: UG
 
 # Troubleshooting and FAQ
 
-## BlinkBinaries are missing
+## Blink files are missing
 
 <table>
 <th style="font-size:14px" width="100px">Exception</th>
-<th style="font-size:14px">BlinkBinaries are missing</th>
+<th style="font-size:14px">Blink files are missing</th>
 <tr>
 <th style="font-size:14px" width="100px">Reason
 </th>
-<td>When BlinkBinaries are not available in the <i>BlinkPath </i> mentioned location.
+<td>The exception may occur if the <i>'runtimes'</i> folder is not copied correctly from the NuGet folder.
 </td>
 </tr>
 <tr>
 <th style="font-size:14px" width="100px">Solution</th>
 <td>
-Set the path of the BlinkBinaries folder to the BlinkPath property of BlinkConverterSettings.
+Ensure that upon NuGet package installation and building, the 'runtimes' folder is automatically copied.
```

Comment:
Ensure that the runtimes folder is copied properly to bin folder of the application from NuGet package location.